spec/models/target_spec.rb

```diff
@@ -8,6 +8,22 @@
     it { should validate_presence_of(:latitude) }
     it { should validate_presence_of(:longitude) }
     it { should validate_presence_of(:user) }
+
+    context 'when a user has 10 targets and tries to create another one' do
+      let!(:user_1) { create :user }
+      it 'should not be able to do so' do
+        targets = create_list(:target, 10, user: user_1)
```

Comment:
Include the list of targets in a let under the context. Remove it from here and from the test below.

```
context 'when the user has already reached the target count limit' do
    let!(:user_1) { create :user }
    let!(targets) { create_list(:target, 10, user: user_1) }
```